.template/addons/phrase/spec/codebase/codebase_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+insert_into_file 'spec/codebase/codebase_spec.rb', before: /end\Z/ do
+  <<~RUBY.indent(2)
+
+    # rubocop:disable RSpec/ExampleLength
+    it 'does not offend Phrase Pull configuration' do
+      locale_tags = Dir[File.expand_path(Rails.root.join('config', 'locales', '*.yml'))]
+                    # ['campaign.en.yml', 'article.en.yml', 'campaign.th.yml', 'en.yml']
```

Comment:
Should we move this comment after the statement, from that, I think it's easier to read.

statement then comment `instead of` the comment before the statement.

Reply:
Do you mean like this? 83eef4d 👀